release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -220,6 +237,12 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Language Server
+
+- Introduced new `Extract to local variable`code action
```

Comment:
What about `Extract to function` code action? https://github.com/ballerina-platform/ballerina-lang/issues/36570